cardano-node/src/Cardano/CLI/Run.hs

```diff
@@ -276,16 +285,13 @@ runCommand co (ToVerification
     . Builder.toLazyText . Crypto.formatFullVerificationKey . Crypto.toVerification
     =<< readSigningKey co skF
 
-runCommand co@CLIOps{..}
-           (Redelegate protoMagic epoch
-            skF vkF (NewCertificateFile certF)) = do
+runCommand co (IssueDelegationCertificate protoMagic epoch skF vkF (NewCertificateFile certF)) = do
```

Comment:
I'm always being chided by Duncan and Marcin, when I break much further beyond 80 lines.. : -)